examples/tutorials/SMPC_Protocols_Explanation.ipynb

```diff
@@ -0,0 +1,390 @@
+{
```

Comment:
<p>Might be interesting to point out that indeed FSS doesn't require lots of communication rounds. But is computationally more intensive than SecureNN (because we call a PRG <code>G</code> many times)</p><p> </p><p>For Dopout, i guess we could implement it, just no one took the time to do so :) </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/OpenMined/PySyft/pull/4635/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>

Reply:
<p>Erratum: we support dropout for secureNN + FSS</p>

Reply:
<p>Alright perfect, I'll add that. </p>